frontend/public/components/utils/resource-log.jsx

```diff
@@ -45,6 +50,33 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(_.sortBy(podLogLinks, 'metadata.name'), (link, i) => {
+        const namespace = resource.metadata.namespace;
+        const namespaceFilter = link.spec.namespaceFilter;
+        const hrefTemplate = link.spec.hrefTemplate;
+        const labels = resource.metadata.labels;
+        // if namespaceFilter, do not output link if namespace does not match namespaceFilter regex
+        if (namespaceFilter && namespace.search(namespaceFilter) === -1) {
+          return null;
+        }
+        const compiled = _.attempt(() => _.template(hrefTemplate));
```

Comment:
Adding @spadgett's comment back after code changes:  
> If we can't find a way to prevent code execution with _.template, we could fall back to...
```
template
  .replace('${name}', encodeURIComponent(resource.metadata.name))
  .replace('${container}', encodeURIComponent(containerName))
  // ...
```